### ISSUE TITLE
Missing config file for reproducing results

### ISSUE THREAD

**Issue by phigre** [OGGM/g2ti#1]
While trying to play around a bit I stumbled upon the problem, that the file '.g2ti_paths' is missing and therefore the runs are aborted before computing anything. Maybe we could easily fix that or at least specify the format for the config?

**Comment by fmaussion** [comment]
yes, definitely not super close to full reproducibility on this repo. Although this could probably be achieved though. 

Here is the content of the file on my laptop

```
geometry_dir = /home/mowglie/disk/G2TI/data/geometry/
index_file = /home/mowglie/disk/G2TI/data/INDEX_thickness_data.csv
thickness_csv_dir = /home/mowglie/disk/G2TI/data/thickness_csv/
index_file_orig = /home/mowglie/disk/G2TI/data/INDEX_thickness_data.txt
index_ref_file = /home/mowglie/disk/G2TI/data/INDEX_reference_ids.csv
```

**Comment by fmaussion** [comment]
this all links to a data directory which contains a mix of original G2TI data and slightly post-processed ones. I have to check the license of this data (got it from Daniel)

**Comment by fmaussion** [comment]
What I would *rather* do though is integrate the original GlaThiDa 2 data in the OGGM framework (like we do for WGMS data) and propose test cases to address the problem of global glacier ice thickness. This would be helpful for you and Matteo. 

We could tackle this in a dedicated paper I want to write soon. In the mean time I'll send you a link to the data in private.
